packages/vat-data/package.json

```diff
@@ -23,7 +23,6 @@
     "@agoric/base-zone": "^0.1.0",
     "@agoric/store": "^0.9.2",
     "@agoric/swingset-liveslots": "^0.10.2",
-    "@agoric/vow": "^0.1.0",
```

Comment:
Looks like a drive-by, since it wasn't needed by vat-data anyway. But until removed, the new devDependence of @agoric/vow on @agoric/zone, for durable zone testing, created an dependency cycle.